index.html

```diff
@@ -1 +1,31 @@
-index
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../assets/css/index.css">
+    <title>Clickable List</title>
+</head>
+
+<body>
+    <ul class="item-list">
+        <li class="clickable-item">
+            <a href="../pages/start.html">
+                <img src="../res/1_start_image.jpg" alt="Item 1">
+                <span>Item 1</span>
+            </a>
+        </li>
+        <li class="clickable-item">
+            <a href="#">
+                <img src="../res/sample_image.jpg" alt="Item 2">
+                <span>Item 2</span>
+            </a>
+        </li>
+        <!-- Add more items as needed -->
+    </ul>
```

Comment:
I think we should make this with javascript code instead of hard-coding

Reply:
We should, but as I have written above, this is a placeholder...